README.md

```diff
@@ -343,6 +343,92 @@ $ plextraktsync --server=Example1 info
+
+```yml
+  Example1:
+    token: ~
+    urls:
+      - http://localhost:32400
+    config:
+      sync:
+        plex_to_trakt:
+          collection: true
+        trakt_to_plex:
+          liked_lists: false
+```
+Using `sync` in a server config overrides the global sync-config in `config.yml`.
+
+This can also be used to have different configs between different libraries. To be able to do this you specifiy the number of servers you need (most likely equal to the number of different config setups you need). For example:
+
+```yml
+  Example1:
+    token: ~
+    urls:
+      - http://localhost:32400
+    config:
+      libraries:
+        - "Movies"
+      sync:
+        plex_to_trakt:
+          ratings: true
+          watched_status: true
+        trakt_to_plex:
+          ratings: true
+          watched_status: true
+  Example2:
+    token: ~
+    urls:
+      - http://localhost:32400
+    config:
+      libraries:
+        - "TV Shows"
+      sync:
+        plex_to_trakt:
+          ratings: true
+          watched_status: false
+        trakt_to_plex:
+          ratings: true
+          watched_status: false
+```
+The above config would make it so that the "Movies" library syncs both ratings and watched status, while the "TV Shows" library only syncs ratings.
+To then run the sync you need to specify `--server Example1` or `--server Example2` to run the sync for that specific server. Running the sync command without it will result in running the sync using the default config.
+
+If you want to run these jobs using `ofelia`, you can do so by running something similar to this in your `docker-compose.yml`:
+```yml
+  plextraktsync:
+    image: ghcr.io/taxel/plextraktsync
+    command: sync
+    container_name: plextraktsync
+    profiles: ["schedule"]
+    volumes:
+      - /configs/mediarr/plextraktsync:/app/config
+    environment:
+      - PUID=1000
+      - PGID=1000
+    depends_on:
+      - plex
+  scheduler:
+    image: mcuadros/ofelia:latest
+    container_name: scheduler
+    command: daemon --docker
+    restart: unless-stopped
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock:ro
+    labels:
+      ofelia.job-run.plextraktsync.schedule: "0 6,18 * * *"
+      ofelia.job-run.plextraktsync.container: "plextraktsync"
+      ofelia.job-run.plextraktsync.command: "--server 'Example1' sync"
+      ofelia.job-run.plextraktsync2.schedule: "0 12,0 * * *"
+      ofelia.job-run.plextraktsync2.container: "plextraktsync"
+      ofelia.job-run.plextraktsync2.command: "--server 'Example2' sync"
+```
+If you are running only one PlexTraktSync container, you need to make sure that the two jobs Ofelia jobs don't run at the same time. Depending on how long it takes for the job to run on your server you might have to keep the schedule of the two jobs seperated with a few minutes or a few hours. If you have two different PlexTraktSync containers in your docker compose, you can run them at the same time.
```

Comment:
```suggestion
If you are running only one PlexTraktSync container, you need to make sure that the two jobs Ofelia jobs don't run at the same time. Ofelia skips scheduling a new job run if the previous job is still running.

  Depending on how long it takes for the job to run on your server you might have to keep the schedule of the two jobs seperated with a few minutes or a few hours. If you have two different PlexTraktSync containers in your docker compose, you can run them at the same time.
```

you still failed to describe the reasoning why overlapping must be avoided.

Reply:
lol true.